features/api/application_plans/metrics_methods_limits_rules.feature

```diff
@@ -0,0 +1,125 @@
+@javascript
+Feature: Application plan Metrics, Methods, Limits & Pricing Rules
+
+  Background:
+    Given a provider is logged in
+    And a product "Dice rolls"
+    And the provider has "service_plans" allowed
+    And the provider has "account_plans" allowed
+    And the following application plan:
+      | Product    | Name | Default |
+      | Dice rolls | Free | true    |
+    # And the provider has plans ready for signups
+    And the product has the following metrics:
+      | Friendly name    |
+      | Single rolls     |
+      | Repeated rolls   |
+      | Complex rolls    |
+      | Custom dice roll |
+
+  Scenario: Pricing rules are hidden when plan is free
```

Comment:
Rather than "plan is free" isn't it "billing is disabled for provider"?
I think "plan is free" is not correct.